src/AdvicesContainer.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react";
+
+import Button from "./Button";
+
+import Advice from "./Advice";
+
+const URL = "https://api.adviceslip.com/advice";
+
+const AdvicesContainer = () => {
+  const [advices, setAdvices] = useState({});
+
+  const [error, setError] = useState(null);
+  const [isDisabled, setDisabled] = useState(false);
+
+  const getAdvice = async () => {
+    await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
Since you are doing a real request, do you need additional timeout in your request?